website/scripts/javadocs.sh

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
-#set -e TODO: figure out why this breaks CI and re-enable (https://github.com/twitter/heron/issues/766)
-
+set -e
```

Comment:
I'm thinking maybe it's because we should first build and then ``make site``? see [ci.sh](https://github.com/twitter/heron/blob/master/scripts/travis/ci.sh)